### PR TITLE
Geometry.fromBufferGeometry -> copy weights and morphs data

### PR DESCRIPTION
**Pull request by damienmortini**
Copy weights and morphs data to Geometry from source BufferGeometry to keep animations.

### CODE REVIEW

**Comment by Mugen87** [comment]
We already had two PRs (#12572 and #10729), none of them were accepted. I feel this is a `Won't fix`.

BTW: What is your use case for converting a `Geometry` to `BufferGeometry`?

File: src/core/Geometry.js [hunk 1]
```diff
@@ -260,6 +263,65 @@ Geometry.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
...
+
+			scope.skinWeights.push( new Vector4( skinWeights[ k ], skinWeights[ k + 1 ], skinWeights[ k + 2 ], skinWeights[ k + 3 ] ) );
+
+		}
+
+		var morphAttributes = geometry.morphAttributes;
+
+		if ( morphAttributes.position ) {
+
+			for ( const morphTargetAttribute of morphAttributes.position ) {
```
Comment (Mugen87) [inline src/core/Geometry.js]: Please don't use `const` and `let`. Also avoid `for of` loops.

**Review by Mugen87 — CHANGES_REQUESTED** [review]

File: src/core/Geometry.js [hunk 2]
```diff
@@ -260,6 +263,65 @@ Geometry.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 			}
 
+		}
```
Comment (Mugen87) [inline src/core/Geometry.js]: add a blank line here

**Comment by damienmortini** [comment]
Just fixed these, thanks!

**Review by damienmortini — COMMENTED** [review]
Updated requested changes

File: src/core/Geometry.js (hunk 2 above)
Comment (damienmortini) [reply src/core/Geometry.js]: All done thanks

File: src/core/Geometry.js (hunk 1 above)
Comment (damienmortini) [reply src/core/Geometry.js]: Ok

**Review by Mugen87 — APPROVED** [review]

**Pull request closed without merging**